src/Login.js

```diff
@@ -1,30 +1,28 @@
-import React from 'react';
-import Stytch from '@stytch/stytch-js'
+import React from "react";
+import Stytch from "@stytch/stytch-js";
 
 const Login = () => {
-	React.useEffect(() => {
-		// Initialize Stytch.js with your public token. You can find this in your Stytch dashboard under API Keys.
-		var STYTCH_PUBLIC_TOKEN = "public-token-test-111111111";
-		var stytch = Stytch(STYTCH_PUBLIC_TOKEN, {
-			onEvent: (response) => {
-				console.log(response);
-			},
-			onSuccess: (response) => {
-				// Handle a successfully sent magic link
-				console.log(response);
-			},
-			onError: (response) => {
-				console.log(response);
-			}
-		});
-		var style = {
-			fontFamily: '"Helvetica New", Helvetica, sans-serif',
-			textAlign: 'center',
-			buttonColor: '#106ee9',
-			fullWidth: true,
-			inputTextColor: '#090909'
-		};
-		var loginOrCreateUserConfig = {
+  React.useEffect(() => {
+    // Initialize Stytch.js with your public token. You can find this in your Stytch dashboard under API Keys.
+    var STYTCH_PUBLIC_TOKEN = "public-token-test-111111111";
+    var stytch = Stytch(STYTCH_PUBLIC_TOKEN, {
+      onEvent: (response) => {
+        console.log(response);
+      },
+      onSuccess: (response) => {
+        // Handle a successfully sent magic link
+        console.log(response);
+      },
+      onError: (response) => {
+        console.log(response);
+      },
+    });
+    var style = {
+      fontFamily: '"Helvetica New", Helvetica, sans-serif',
+      button: { color: "#106ee9" },
```

Comment:
@nathan-stytch should this be textColor? Did you run this app locally to make sure it works?

Reply:
or this is backgroundColor I think